CONTRIBUTING.md

```diff
@@ -1,18 +1,102 @@
+- [ ] Requires no OpenStreetMap knowledge!
+
+As you probably noticed, you can choose "Cannot answer" in StreetComplete and thus leave a note by
+
+You can help with [processing OSM notes opened by StreetComplete users](https://ent8r.github.io/NotesReview/?query=StreetComplete&limit=100&start=true). (You can also do it without that special tool, but it helps.) When doing so, you can also find systematic problems or misunderstandings of user's when solving StreetComplete quests.
+
+If you find such user experience problems, please report them back in the [issue tracker of StreetComplete](https://github.com/westnordost/StreetComplete/issues). Do not forget to add links to examples, where you found them.
+
+## Joining discussions
+
+- [x] Requires no programming knowledge!
+- [x] Requires no OpenStreetMap knowledge!
+
+You can always join in discussions in issues for new quests or similar stuff. Often local knowledge or so is required, so we can estimate whether a quest can be asked in different regions.
+OpenStreetMap (and StreetComplete) works, because of a diverse community all over the world.
+
+[Especially look at the issues tagged with "help wanted"](https://github.com/westnordost/StreetComplete/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22).
+
+## Testing and reporting issues
+
+- [x] Requires no programming knowledge!
+- [x] Requires no OpenStreetMap knowledge!
+
+If you experience problems, crashes or a quest is not clear to you, feel free to open an issue for that. Remember to open one issue _for one matter_, so do not open one issue "I have found several problems", but one for each problem.
+
+1. If you have questions, remember to [read the FAQ](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ) first.
+2. Look [whether your issue has already been reported](https://github.com/westnordost/StreetComplete/issues).
+3. Now you can open a new issue. Give it a meaningful title and describe the problem. Do mention the StreetComplete and Android version, if it is a technical bug.
+
+### Map style issues
+
+Note that the map style used by StreetComplete is developed as a [separate project](https://github.com/ENT8R/streetcomplete-mapstyle), so please report issues related to it there.
+
+### Suggesting new quests
+
+We appreciate all quest ideas, you and others have, but some ideas cannot be added into the app, due to various design decicions and requirements. This is required in order to make a great app for all users of StreetComplete, whose contributions actually help the OpenStreetMap project. A quest, which nobody can solve and where users potentially choose wrong options and add wrong data to OpenStreetMap hurts not only OSM, but also this app.
+
+What you need to do:
+
+1. Search whether the quest you want to suggest is [already implemented in StreetComplete](https://wiki.openstreetmap.org/wiki/StreetComplete/Quests) or [already suggested in the issue tracker](https://github.com/westnordost/StreetComplete/issues).
```

Comment:
The issue tracker hyperlink should include the new quest label.

Reply:
Good point, I also thought this (i.e. it should link to [this search](https://github.com/westnordost/StreetComplete/issues?q=is%3Aissue+is%3Aopen+label%3A%22new+quest%22)), but @westnordost uses the quests label also as a kind of "this is a quest I have approved" label. So often, closed/declined quests (e.g.. [this](https://github.com/westnordost/StreetComplete/issues/989), [this](https://github.com/westnordost/StreetComplete/issues/956)) do not get the label.
However, users should of course also find these declined quests, so they don't open duplicates. Basically that's all what this point is about.

IMHO this is indeed a good idea, but currently not possible. It's something @westnordost has to do. (I mean, if I could, I'd happily just go through the issues and label them as quests.)
So as it cannot be fixed in this PR, as I assume, I open a new issue for it. (also so we don't forget it :wink:)